frontend/package.json

```diff
@@ -68,16 +68,21 @@
     "@trivago/prettier-plugin-sort-imports": "^4.3.0",
     "eslint": "^8.57.0",
     "eslint-config-prettier": "^9.1.0",
+    "eslint-import-resolver-typescript": "^3.7.0",
+    "eslint-plugin-import": "^2.31.0",
+    "eslint-plugin-jsx-a11y": "^6.10.2",
     "eslint-plugin-prettier": "^5.1.3",
-    "eslint-plugin-react": "^7.34.1",
-    "eslint-plugin-storybook": "^0.8.0",
+    "eslint-plugin-react": "^7.37.3",
+    "eslint-plugin-storybook": "^0.11.2",
+    "globals": "^15.14.0",
     "husky": "^9.0.11",
     "lint-staged": "^15.2.2",
     "prettier": "3.2.5",
     "prop-types": "^15.8.1",
     "storybook": "^8.0.10",
     "storybook-addon-mock": "^5.0.0",
     "storybook-addon-module-mock": "^1.3.4",
+    "typescript-eslint": "^8.19.0",
```

Comment:
was confusing to track down, but this is a meta package that has the plugin and parser. CRA itself also depends on it, but an older version of course.